content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -215,7 +215,40 @@ Nevertheless, you must update the list of admission controllers.
 
 ### Azure
 
-You must use `ACS-Engine` to deploy your cluster.
+You can deploy a Kubernetes cluster to Azure via `ACS-Engine` or `AKS` which fully supports Istio as long as Kubernetes 1.9.6 or later is specified.
```

Comment:
Please put AKS before ACS-Engine to keep consistent with the installation instruction order